build/images/scripts/install_cni_windows.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = "Stop";
+
+mkdir -force /host/var/run/secrets/kubernetes.io/serviceaccount
+cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/run/secrets/kubernetes.io/serviceaccount/
+mkdir -force /host/k/antrea/etc/
+mkdir -force /host/k/antrea/logs/
+cp /k/antrea/cni/* /host/opt/cni/bin/
+cp /etc/antrea/antrea-agent.conf /host/k/antrea/etc/
+
+cp /etc/antrea/antrea-cni.conflist /host/etc/cni/net.d/10-antrea.conflist
```

Comment:
Nit - in some files you added a new line at the end, some files not.

Reply:
OK. I will leave a new lline for all powershell scripts.